recipes-domd/agl/inc/domd-agl-image.inc

```diff
@@ -26,6 +26,9 @@ SRC_URI_append = " \
     git://git.yoctoproject.org/meta-virtualization;destsuffix=repo/meta-virtualization;branch=morty \
     git://git.yoctoproject.org/meta-selinux;destsuffix=repo/meta-selinux;branch=morty \
 "
+# LLVM Clang section. Needed to build gles-user-module
+XT_QUIRK_BB_ADD_LAYER += " meta-clang"
```

Comment:
why do you need separate SRC_URI_append/XT_QUIRCK_BB_ADD_LAYER, please append the above ones

Reply:
Done